slack-api-model/src/main/java/com/slack/api/model/workflow/WorkflowStepInput.java

```diff
@@ -0,0 +1,47 @@
+package com.slack.api.model.workflow;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Map;
+
+/**
+ * <pre>
```

Comment:
For my knowledge, what is the purpose of the tags included within the docstrings?

Reply:
As we have to make the types of the fields in this class dynamic (e.g., java.lang.Object), I wanted to place some examples taken from the API documents. But we can update or revisit in the future when we have a more comprehensive guide for the types.